edx_plugin/apps.py

```diff
@@ -1,13 +1,71 @@
-"""
-edx_plugin Django application initialization.
-"""
-
 from django.apps import AppConfig
+from edx_django_utils.plugins.constants import (
+    PluginURLs, PluginSettings, PluginContexts
+)
+class EdxPluginConfig(AppConfig):
+    name = 'edx-plugin.edx_plugin'
```

Comment:
Please, double-check this can create an issue - I believe app name must comply with python modules naming convention. Possibly, you need avoid dash symbol here.

Reply:
To control this you may revise LMS logs after devserver restart: if plugins module cannot import something it complains there.